src/kendo/Bar.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["jquery","kendo.all.min","../common/HTMLWidget"], factory);
+    } else {
+        root.kendo_Bar = factory();
+    }
+}(this, function (jquery, kendo, HTMLWidget) {
```

Comment:
Is JQuery actually needed?  Or is it just a dependency of kendo (if so we can probably move this to a shim, or let bower/kendo handle it)?